openwisp_controller/geo/utils.py

```diff
@@ -1,21 +0,0 @@
-from django.conf.urls import url
-
-
-def get_geo_urls(geo_views):
-    return [
-        url(
-            r'^api/v1/device/(?P<pk>[^/]+)/location/$',
-            geo_views.device_location,
-            name='api_device_location',
-        ),
-        url(
-            r'^api/v1/location/geojson/$',
-            geo_views.geojson,
-            name='api_location_geojson',
-        ),
-        url(
-            r'^api/v1/location/(?P<pk>[^/]+)/device/$',
-            geo_views.location_device_list,
-            name='api_location_device_list',
-        ),
-    ]
```

Comment:
This is not the correct place for the urls, so moved it to dedicated `urls.py` file in `api` dir.

Reply:
I think we shouldn't change it. It might have backward compatibility issues.
@nemesisdesign can you please confirm this.

Reply:
Yes you are right Ajay, we should add a shim for backward compatibility.